inst/templates/ad_advs.R

```diff
@@ -0,0 +1,269 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis dataset
+#
+# Input: adsl, vs
+
+
+# Attach/load required packages ----
+library(admiral)
+library(admiralmetabolic)
+library(tibble)
+library(dplyr)
+library(stringr)
+
+
+# Define project options/variables ----
+# Use the admiral option functionality to store subject key variables in one
+# place
+set_admiral_options(subject_keys = exprs(STUDYID, USUBJID))
```

Comment:
```suggestion
```
subject_keys defaults to exprs(STUDYID, USUBJID), we don't need to set this up

Reply:
The reason why its manually set, is too highlight that we will be using this functionality through the template. I feel that this should be kept, but have added some text explaining that the default is STUDYID and USUBJID.

Reply:
I like this as well @AndersAskeland, no problem if we deviate a bit from ADVS in admiral core.